packages/react-components/src/components/SendBox.tsx

```diff
@@ -39,6 +40,33 @@ export interface SendBoxStylesProps extends BaseCustomStyles {
   systemMessage?: IStyle;
 }
 
+/**
+ * Attributes required for SendBox to show file uploads like name, progress etc.
+ * @beta
+ */
+export interface ActiveFileUploads {
+  /**
+   * Unique identifier for the file upload.
+   */
+  id: string;
+
+  /**
+   * Filename extracted from the {@link File} object.
```

Comment:
Incorrect documentation.
It should just be `File name to render`
Because, in the context of SendBox there is no File Object or metdata

Reply:
there seems to be a caveat "if metadata.name is not available" does this suggest this property should be optional?

Reply:
We have file information in metadata as well as file upload object. Metadata might not be available all the time, here we are extracting file info from the file uploads.